### ISSUE TITLE
Mobile friendly responsive design

### ISSUE THREAD

**Issue by zx2c4** [F43nd1r/Acrarium#22]
Right now, if the browser is too narrow, the table gets a bit confused:

![img_20180729_144133](https://user-images.githubusercontent.com/10643/43366390-8c3ba1b8-933d-11e8-82c4-4099a81f2ae0.jpg)

**Comment by F43nd1r** [comment]
Currently, all non-critical UI fixes are on hold. The're likely to get fixed by #13, which is currently blocked by a missing feature in the vaadin-grid-flow upstream.

**Comment by F43nd1r** [comment]
Vaadin flow grid correctly shows a scrollbar on small screens.
